### ISSUE TITLE
Please include screenshots

### ISSUE THREAD

**Issue by PabloCastellano** [jros2300/reservedinstances#11]
Hey there.

I find this project very interesting and I'm curious to know what is the look and feel. I still haven't had the chance to install it. Could you add some screenshots?

**Comment by mrPsycho** [comment]
yes. project looks interesting, but there is no way to get permission from teamlead to test it without any screens (
